cmake/modules/HexagonSDK.cmake

```diff
@@ -97,11 +132,26 @@ function(_get_hexagon_sdk_property_impl
     endif()
 
   elseif(_property STREQUAL "QAIC_EXE")
-    _check_path_exists(
-      "${_hexagon_sdk_root}/ipc/fastrpc/qaic/Ubuntu18/qaic"
-      _qaic_path
-    )
-    set_parent(${_output_variable} "${_qaic_path}")
+    set(_override $ENV{QAIC_PATH_OVERRIDE})
+    if(_override)
+      set_parent(_qaic_path "${_override}")
+    else()
+      _get_ubuntu_version(_uversion)
+      _check_path_exists(
+        "${_hexagon_sdk_root}/ipc/fastrpc/qaic/${_uversion}/qaic"
+        _qaic_path
+      )
+    endif()
+    _check_path_exists("${_qaic_path}" _qaic_path_found)
+    if(NOT _qaic_path_found)
+      message(
+        SEND_ERROR
+        "The qaic executable cannot be found in '${_qaic_path}'. You can set "
+        "the environment variable QAIC_PATH_OVERRIDE to override the automatic "
+        "search."
```

Comment:
IIUC, for most/all other properties handled by `_get_hexagon_sdk_property_impl`, the convention is to set the output variable to `"${_property}-NOTFOUND"` if property value isn't valid.  This is even true for properties handled by `_check_all_paths_exist`.

Would it make sense to handle this property in the same way, rather than calling `message(SEND_ERROR, ...)`?

Reply:
Makes sense.